libraries/chain/authorization_manager.cpp

```diff
@@ -559,7 +559,7 @@ namespace eosio { namespace chain {
 
       }
 
-      if( !allow_unused_keys  || check_but_dont_fail) {
+      if( !allow_unused_keys  && !check_but_dont_fail) {
```

Comment:
Thanks.But `EOS_ASSERT` will only throw the error when the first argument is false. Since `check_but_dont_fail` should never throw the error when it is true,  there is no need to modify at line 548.

Reply:
Oh yes, what was I thinking!